airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java

```diff
@@ -357,8 +357,8 @@ public void testSecondSync() throws Exception {
                     .put("id", 1)
                     .put("currency", "USD")
                     .put("date", "2020-03-31T00:00:00Z")
-                    .put("HKD", 10)
-                    .put("NZD", 700)
+                    .put("HKD", 10.0)
+                    .put("NZD", 700.0)
```

Comment:
`HKD` and `NZD` are typed as `number` in the `catalog`. All other entries have decimals for these two fields. So I'd like to change these values to decimals as well so that the type is consistent.

In Parquet and probably other formats, we need to have strict type mappings, and `number` is mapped to `double`. If these two fields have flipping types, I need to do arbitrary conversions to pass the acceptance test, which seems unnecessary.